vktrace/vktrace_trace/vktrace.cpp

```diff
@@ -390,6 +390,18 @@ int main(int argc, char* argv[]) {
             fflush(stdout);
             g_settings.arguments = NULL;
         } else {
+            // We are not in server mode.
+
+            // Verify trace layer is available.
+            void* traceLayerLibHandle;
+            traceLayerLibHandle = vktrace_platform_open_library(VKTRACE_LIBRARY_NAME(VkLayer_vktrace_layer));
```

Comment:
This will search the system library path, but VK_LAYER_PATH will also need to be searched for this case.  It may be necessary to create an instance to check for the presence of the layer.